tasks/jest/setupAfterEnv.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const toHaveNoViolations = require('./matchers/toHaveNoViolations');
```

Comment:
I see what this is doing, but I don't know why we need to do this 😸

Reply:
What specifically? In this case, we need to extend `expect` so that anywhere in our tests we can call `expect(...).toHaveNoViolations` similar to `expect(...).toEqual(...)`

Reply:
Gotcha! Exactly what I was asking. Thanks 👍

Reply:
With the file name "setup.." I just wanting to make sure I knew what was happening here.

Reply:
Totally fair, let me add a block 👍

Reply:
Added!